partcad/src/partcad/test/cad.py

```
@@ -0,0 +1,24 @@
+
+class CadTest(Test):
+    def __init__(self):
+        super().__init__("cad")
+
+    async def test(self, ctx, shape):
+        wrapped = await shape.get_wrapped()
+        if wrapped is None:
+            pc_logging.error("Failed to get the shape of %s:%s" % (shape.package, shape.name))
+            return
+
+        pc_logging.debug("Passed test: %s: %s:%s" % (self.name, shape.project_name, shape.name))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)